pkgs/tools/system/zcfan/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+# Testing this requires a Thinkpad or the presense of /proc/acpi/ibm/fan
+stdenv.mkDerivation rec {
+  pname = "zcfan";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "cdown";
+    repo = "zcfan";
+    rev = version;
+    hash = "sha256-XngchR06HP2iExKJVe+XKBDgsv98AEYWOkl1a/Hktgs=";
+  };
+
+  doInstallCheck = true;
+
+  configurePhase = ''
+    substituteInPlace Makefile \
+      --replace "/usr/local" $out
+  '';
+
+  buildPhase = ''
```

Comment:
add `runHook preBuild` and `runHook postBuild`

Reply:
buildPhase removed in https://github.com/NixOS/nixpkgs/pull/253579/commits/6409b451bc38bfe3ed3f313308d62b35d35e59d2